community/OM-Operator-Management/cnv-samples/fleet-user-rolebinding-policy/acm-kubevirt-edit-fleet.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: acm-kubevirt-edit-fleet
+  namespace: default
+  annotations:
+    policy.open-cluster-management.io/categories: AC Access Control
+    policy.open-cluster-management.io/controls: AC-3 Access Enforcement
+    policy.open-cluster-management.io/description: This is the virt.k8s.io:edit ClusterRole for all CNV managed clusters
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+spec:
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-rolebinding
+        spec:
+          namespaceSelector:
+            exclude:
+              - kube-*
+            include:
+              - default
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: rbac.authorization.k8s.io/v1
+                kind: ClusterRoleBinding
+                metadata:
+                  name: acm-kubevirt-edit
+                roleRef:
+                  name: kubevirt.io:edit
+                  apiGroup: rbac.authorization.k8s.io
+                  kind: ClusterRole
+                subjects:
+                  - name: jnpacker
+                    apiGroup: rbac.authorization.k8s.io
+                    kind: User
+                  - name: davidvossel
+                    apiGroup: rbac.authorization.k8s.io
+                    kind: User
+            - complianceType: musthave
+              objectDefinition:
+                kind: User
+                apiVersion: user.openshift.io/v1
+                metadata:
+                  name: virtuser01
+                fullName: Virtual User 01
+                groups: null
+            - complianceType: musthave
+              objectDefinition:
+                kind: User
+                apiVersion: user.openshift.io/v1
+                metadata:
+                  name: virtuser02
+                fullName: Virtual User 02
+                groups: null
+          remediationAction: inform
+          severity: high
+  remediationAction: enforce
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: acm-virt-edit-fleet-placement
+  namespace: default
+placementRef:
+  name: openshift-virtualization
+  apiGroup: apps.open-cluster-management.io
+  kind: PlacementRule
```

Comment:
If you update to `Placement`, don't forget to update down here too.

Reply:
yes. Ty